### ISSUE TITLE
Remove mobile code

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-e2e-tests#428]
If we've decided to not continue doing mobile app testing in this repository we can remove all the mobile code and dependencies and update the README as well

**Comment by chenchaoyi** [comment]
If this has been decided (it looks like XCUItest has been chosen), mind if I take this ?
It'll help me to get familiar with the repo structure too.

**Comment by alisterscott** [comment]
@chenchaoyi: absolutely, feel free. Thanks

**Comment by hoverduck** [comment]
@chenchaoyi - I had actually already done this work under [this branch](https://github.com/Automattic/wp-e2e-tests/tree/update/remove-iOS-tests), but I held off on creating a PR or merging it until you'd started on your trial so as not to influence what you put together for Android.  And then I forgot about it 😄 

But it is a good excuse for you to poke around this repo, so I'll leave it to you to merge.

**Comment by chenchaoyi** [comment]
@hoverduck lucky I asked here :)

I'll get down to this after #438 then

**Comment by chenchaoyi** [comment]
Close as #446 is merged.
